---
permalink: /
layout: default
--- 

<div class="grid" style="margin: 30px">

	<h3>Research</h3>
	<br>
	My research is in statistical genetics. My research interests are in developing generic statistical methods and efficient computational tools for understanding biological questions, especially those related to large-scale human genetics data. Currently, I am working on gene-environment interactions, genetic risk prediction, and multi-ancestry analysis.

	<h3>Puclications and Projects</h3>


	<h4>Preprint</h4>

	<br>
	<div class="grid" style="margin-bottom: 10px;">
		<smallpd><strong>Statistical Methods for Gene-environment Interaction Analysis.</strong>  
			<br>
			<smallpd><u>Miao J.</u>, Wu Y., Lu Q. (2023).</smallpd>
			<br>
			<smallpd> <strong>Submitted. (Review)  </strong></smallpd> 
	</div>

	<br>
	<br>
	<div class="grid" style="margin-bottom: 10px;">
		<smallpd><strong>The impact of genomic variation on function (IGVF) consortium</strong>
			<br>
			<smallpd>IGVF Consortium (2023)</smallpd>  
			<br>
			<smallpd> <strong>Submitted. (preprint available on arXiv)</strong></smallpd> 
			<br>
			<smallpd>
				<a href="https://arxiv.org/abs/2307.13708">Preprint</a>
			</smallpd>
	</div>

	<br>
	<br>
	<div class="grid" style="margin-bottom: 10px;">
		<smallpd><strong> Reimagining Gene-Environment Interaction Analysis for Human Complex Traits</strong>  
			<br>
			<smallpd><u>Miao J.</u>, Song G., Wu Y., Hu J., Wu Y., Basu S., Andrews J., Schaumberg K., Fletcher J., Schmitz L., Lu Q. (2022).</smallpd>
			<br>
			<smallpd> <strong>Submitted. (preprint available on bioRxiv)  </strong></smallpd> 
			<br>
			<smallpd>
				<a href="https://www.biorxiv.org/content/10.1101/2022.12.11.519973v1">Preprint</a> |
				<a href="https://github.com/qlu-lab/PIGEON">Software</a> |
			</smallpd>
			<br>
			<br>
			<smallpd>
			--We present a unified theory and framework, called PIGEON, for modeling polygenic GxE effects for complex traits. It allows us to define the estimands of interest and to establish the connection and distinction between different methods in GxE inference. Motivated by our theory, we have also developed an innovative approach to estimate GxE interactions using genome-wide summary data. It is unbiased, computationally efficient, robust to sample overlap, heteroscedasticity, and gene-environment correlation.
			</smallpd>
	</div>

	
	<br>
	<br>
	<div class="grid" style="margin-bottom: 10px;">
		<smallpd><strong> Neurogenetic Mechanisms of Risk for ADHD: Examining Associations of Functionally-Annotated Polygenic Scores and Brain Volumes in a Population Cohort.</strong>  
			<br>
			<smallpd>He, Q., Keding, T., Zhang, Q., <u>Miao J.</u>, Herringa, R., Lu, Q., Travers, B., & Li, J. J. (2022).</smallpd>
			<br>
			<smallpd> <strong>Submitted. (preprint available on medRxiv)  </strong></smallpd> 
			<br>
			<smallpd>
				<a href="https://www.medrxiv.org/content/10.1101/2022.12.12.22283356v1">Preprint</a>
			</smallpd>
	</div>

	<br>
	<br>
	<div class="grid" style="margin-bottom: 10px;">
		<smallpd><strong>Optimizing and benchmarking polygenic risk scores with GWAS summary statistics.</strong>
			<br> 
			<smallpd>Zhao, Z., Gruenloh, T., Wu, Y., Sun, Z., <u>Miao J.</u>, Wu, Y., Song, J., & Lu, Q. (2022).</smallpd>  
			<br>
			<smallpd> <strong>Submitted. (preprint available on bioRxiv)  </strong></smallpd> 
			<br>
			<smallpd>
				<a href="https://www.biorxiv.org/content/10.1101/2022.10.26.513833v2">Preprint</a> | 
				<a href="https://github.com/qlu-lab/PUMAS">Software</a>
			</smallpd>
	</div>

	<h4>2023</h4>

	<br>
	<div class="grid" style="margin-bottom: 10px;">
		<smallpd><strong>Decomposing heritability and genetic covariance by direct and indirect effect paths.</strong>
			<br>
			<smallpd>Song J., Zou Y., Wu Y., <u>Miao J.</u>,  Yu Z., Fletcher J., Lu Q. (2023).</smallpd> 
			<br>
			<smallpd> <strong>PLOS Genetics  </strong></smallpd> 
			<br>
			<smallpd>
				<a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1010620">Journal</a>
			</smallpd>
	</div>

	<br>
	<div class="grid" style="margin-bottom: 10px;">
		<smallpd><strong>Neuropathology-based APOE genetic risk score better quantifies Alzheimer's risk.</strong>
			<br>
			<smallpd>Deming Y., Vasiljevic E., Morrow A., <u>Miao J.</u>,  Van Hulle C., Jonaitis E., Ma Y., Whitenack V., Kollmorgen G., Wild N., Suridjan I., Shaw L., Asthana S., Carlsson C., Johnson S., Zetterberg H., Blennow K., Bendlin B., Lu Q., Engelman C., the Alzheimer's Disease Neuroimaging Initiative (2023).</smallpd>  
			<br>
			<smallpd> <strong>Alzheimer's & Dementia  </strong></smallpd> 
			<br>
			<smallpd>
				<a href="https://alz-journals.onlinelibrary.wiley.com/doi/10.1002/alz.12990">Journal</a> | 
				<a href="https://www.medrxiv.org/content/10.1101/2022.10.12.22280874v1">Preprint</a>
			</smallpd>
	</div>


	<br>
	<div class="grid" style="margin-bottom: 10px;">
		<smallpd><strong>Quantifying portable genetic effects and improving cross-ancestry genetic prediction with GWAS summary statistics.</strong>
			<br>
			<smallpd><u>Miao J.*</u>, Guo H.*, Song G., Zhao Z., Hou L.<sup>&dagger;</sup>, Lu Q.<sup>&dagger;</sup> (2023)</smallpd>
			<br>
			<smallpd> <strong>Nature Communications  </strong></smallpd> 
			<br>
			<smallpd>
				* equal contribution | 
				<a href="https://www.nature.com/articles/s41467-023-36544-7">Journal</a> | 
				<a href="https://github.com/qlu-lab/X-Wing">Software</a>
			</smallpd>
			<br>
			<br>
			<smallpd>
			--We introduce a cross-population genetic risk prediction framework, called X-Wing, that (1) quantifies cross-population local genetic correlation (2) incorporates it as an annotation into a Bayesian framework which amplifies correlated SNP effects between populations through annotation-dependent statistical shrinkage (3) combines multiple population-specific PRS to further improve prediction accuracy while using GWAS summary data alone as input.
			</smallpd>
	</div>


	<h4>2022</h4>

	<br>
	<br>
	<div class="grid" style="margin-bottom: 30px;">
		<smallpd><strong>A quantile integral linear model to quantify genetic effects on phenotypic variability.</strong>
			<br> 
			<smallpd><u>Miao J.</u>, Lin Y., Wu Y., Zheng B., Schmitz L., Fletcher J., Lu Q. (2022).</smallpd>
			<br>
			<smallpd> <strong>Proceedings of the National Academy of Sciences (PNAS)  </strong></smallpd> 
			<br>
			<smallpd>
				<a href="https://www.pnas.org/doi/abs/10.1073/pnas.2212959119">Journal</a> |
				<a href="https://github.com/qlu-lab/QUAIL">Software</a> 
			</smallpd>
			<br> <!-- Added one more break for spacing -->
			<smallpd>
			&nbsp;&nbsp;&nbsp;&nbsp; - Distinguished Student Paper Award from the Section on Statistical Genomics and Genetics of the American Statistical Association (ASA) 2022.
			<br>
			</smallpd>
			<smallpd>
			&nbsp;&nbsp;&nbsp;&nbsp; - Reviews's Choice Award from American Society of Human Genetics Meeting (ASHG) Meeting 2021.
			</smallpd>
			<br>
			<br>
			<smallpd>
			--We propose a unified statistical framework, called QUAIL, for estimating genetic effects on the variability of quantitative traits and for prioritizing genetic variants involved in interactions. QUAIL constructs a quantile-integral phenotype that aggregates information from all quantile levels, makes no assumptions about the distribution of the phenotype, and accounts for confounding effects at both the trait level variability.
			</smallpd>	
	</div>

	<br>
	<br>
	<div class="grid" style="margin-bottom: 30px;">
		<smallpd><strong>Identifying genetic loci associated with complex trait variability.</strong>
			<br>
			<smallpd><u>Miao J.</u>, Lu Q. (2022).</smallpd>
			<br>
			<smallpd> <strong>Handbook of Statistical Bioinformatics (2nd Edition). Springer. (Book Chapter)  </strong></smallpd> 
			<br>
			<smallpd>
				<a href="https://link.springer.com/chapter/10.1007/978-3-662-65902-1_13">Book Chapter</a> 
			</smallpd>	
	</div>



	<h4>2021</h4>

	<br>
	<div class="grid" style="margin-bottom: 30px;">
		<smallpd><strong>The socioeconomic gradient in epigenetic aging clocks: evidence from the Multi-ethnic Study of Atherosclerosis and the Health and Retirement Study.</strong>
			<br>
			<smallpd>Schmitz L., Zhao W., Ratliff S., Goodwin J., <u>Miao J.</u>, Lu Q., Guo X., Taylor K., Ding J., Liu Y., Levine M., Smith J. (2021).</smallpd>
			<br>
			<smallpd> <strong>Epigenetics  </strong></smallpd> 
			<br>
			<smallpd>
				<a href="https://www.tandfonline.com/doi/full/10.1080/15592294.2021.1939479?journalCode=kepi20">Journal</a> 
			</smallpd>	
	</div>

	<br>
	<div class="grid" style="margin-bottom: 30px;">
		<smallpd><strong>The impact of late-career job loss and genetic risk on body mass index: evidence from variance polygenic scores.</strong>
			<br>
			<smallpd>Schmitz L., Goodwin J., <u>Miao J.</u>, Lu Q., Conley D. (2021).</smallpd>
			<br>
			<smallpd> <strong>Scientific Reports  </strong></smallpd> 
			<br>
			<smallpd>
				<a href="https://www.nature.com/articles/s41598-021-86716-y">Journal</a> 
			</smallpd>	
	</div>

	<h3>Academic Services</h3>
	<div class="grid" style="margin-bottom: 30px;">
		<smallpd>
			<ul>
			  <li><strong>Reviewer:</strong>
				Behaviour Genetics, BMC Genomics
			  </li>
			  <li><strong>Mentor:</strong>
				- Gefei Song, Undergraduate Researcher at UW-Madison '22, Winner of Hilldale Undergraduate/Faculty Research Award
				- Yixuan Wu, Undergraduate Researcher at UW-Madison '23, Winner of Hilldale Undergraduate/Faculty Research Award
			  </li>
			</ul>
		</smallpd>
	</div>

</div>




